api/pkg/transformer/server/grpc/server_test.go

```diff
@@ -598,7 +598,7 @@ func TestUPIServer_PredictValues_WithMockPreprocessAndPostprocessHandler(t *test
 				incomingReq = (*upiv1.PredictValuesRequest)(req)
 			}
 			clientMock.On("PredictValues", mock.Anything, incomingReq, mock.Anything).Return(tt.expectedModelOutput, tt.modelErr)
-			noOpTracer := trace.NewNoopTracerProvider()
+			noOpTracer := noop.NewTracerProvider()
```

Comment:
This change is pretty straightforward - we just need to replace the deprecated `trace.NewNoopTracerProvider()` with `noop.NewTracerProvider()`.